### PR TITLE
Merge async branch with dev

### PR DESCRIPTION
**Pull request by cra1gg**
Finished async changes, need to move on to word -> phone

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -13,60 +13,38 @@ const firebaseConfig = {
...
+    var ref = database.ref();
 
-/*     
-    console.log("begin");
-    var leadsRef = database.ref('mappings');
-    leadsRef.on('value', function(snapshot) {
-         snapshot.forEach(function(childSnapshot) {
-         var childData = snapshot.node_.children_.root_.value.value_;
-         console.log("snapshot.node_.children_.root_.value.value_: ", snapshot.node_.children_.root_.value.value_)
+    var word1 = ref.child('mappings').orderByChild('number').equalTo(first_num).once("value")
```
Comment (shmkane) [inline js/main.js:34]: does this need to be ordered by child? or can we directly do .equalTo?

File: js/main.js [hunk 2]
```diff
@@ -13,60 +13,38 @@ const firebaseConfig = {
...
+    var word2 = ref.child('mappings').orderByChild('number').equalTo(second_num).once("value")
```
Comment (shmkane) [inline js/main.js:35]: Same thing here ^

File: js/main.js (hunk 1 above)
Comment (cra1gg) [reply js/main.js:34]: unfortunately as far as I know, this is the equivalent of equalTo in firebase. the syntax looks weird but it achieves the same thing

File: js/main.js (hunk 2 above)
Comment (cra1gg) [reply js/main.js:35]: answered above

**Pull request merged**